### DISCUSSION TITLE
Red box w/ exclamation point on Tampermonkey icon - No Info

### DISCUSSION THREAD

**Discussion by inconspiciousfbivan** [Tampermonkey/tampermonkey#1580] [Q&A]
I have this box sitting on my Tampermonkey add-on icon, and it will not go away. I notice this happens quite often, but no matter where I go in the add-on, I cannot find what Tampermonkey is trying to alert me of or what the problem is. I just updated both the script it runs, and the add-on itself and it is still complaining. Closing/opening the browser does nothing. The browser itself was updated as well. The script it is running is working just fine, so what could this possibly be? Trying to find out what the problem or what the add-on is trying to tell me is impossible for a novice and someone who does not do any coding. I have even checked the Tampermonkey FAQ to no avail, and not even searching online can find anyone asking a similar question.

**Comment by derjanb** [comment]
What is your browser and version?
